synapse/lib/types.py

```diff
@@ -624,6 +624,7 @@ class Hex(Type):
 
     _opt_defs = (
         ('size', 0),  # type: ignore
+        ('pad', False),
```

Comment:
do we want the name to indicate that this is zero padding? (just thinking about if we potentially add a "sign extension" option as well in the future)

Reply:
Yeah, maybe. The ticket says `0pad` but having options that start with a number doesn't feel right. Maybe `zpad` or `zeropad`? `zlpad` to indicate leading zeroes? Other ideas?